docs/getting-started/building-bpfman.md

```diff
@@ -200,6 +200,22 @@ sudo dnf install llvm-devel clang-devel elfutils-libelf-devel
 sudo apt install clang lldb lld libelf-dev gcc-multilib
 ```
 
+### Install libssl Development Library
+
+`apt` based OS:
+
+```console
+sudo apt install libssl-dev
+```
+
+### Install bpf Helper Header Files
```

Comment:
This install is described in https://bpfman.io/main/getting-started/example-bpf/#building-locally, which is linked from https://bpfman.io/main/getting-started/example-bpf-local/#running-example-programs

Do you think we should move the examples specific install packages to this file? Was trying to keep the examples stuff separate, but may not be the best practice.

Reply:
I have to admit that I completely missed the "Building Locally" for building the examples.

![left sidebar](https://github.com/bpfman/bpfman/assets/6920158/5ca90ebe-d87e-4a13-b572-c17817155e46)

In its current form, the documentation first describes how to deploy the example programs, but not how to build them. This is only shown in the last-but-one item, and then it's not even clear from the sidebar that it includes instructions for the needed building.

I'm open to discussing this further, as my gut feeling is that this part of the documentation needs refactoring: at least the example building instructions should come before the deployment examples. Maybe we should split the building instructions from https://bpfman.io/main/getting-started/example-bpf/#examples-in-kubernetes-deployment, keeping the major rest as is; the split part then could move into its own new "Building Examples" that comes before https://bpfman.io/main/getting-started/example-bpf-local/

Reply:
I agree with your assessment. We have already been discussing other changes to the docs (see comments in #1053). So let's punt on this for your PR and we will address it with some of the overall docs rework.

Reply:
Shall I keep this change, or drop it?